js/web/lib/wasm/jsep/webgpu/ops/batch-norm.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {env} from 'onnxruntime-common';
+
+import {TensorView} from '../../tensor-view';
+import {ShapeUtil} from '../../util';
+import {AttributeWithCacheKey, createAttributeWithCacheKey} from '../attribute-with-cache-key';
+import {ComputeContext, ProgramInfo} from '../types';
+
+import {createTensorShapeVariables, enableShapesUniforms, getMaxComponents, inputVariable, outputVariable, ShaderHelper} from './common';
+
+export interface BatchNormAttributes extends AttributeWithCacheKey {
+  readonly epsilon: number;
+  readonly momentum: number;
+  readonly spatial: boolean;
+  readonly trainingMode: boolean;
+  readonly format: 'nhwc'|'nchw';
```

Comment:
Maybe using upper cases for the format values?

I found that there are a few places for the tensor layout using lower cases ("nhwc") and some others using upper cases ("NHWC").

WebNN is currently using lower cases, and existing WebGPU attributes are using upper cases.

Reply:
let me know if you think lower case is better, so that I can make a change to make all JS string literal for tensor layout to lower case.

Reply:
Change it to upper case to align with the current webgpu implementation.
In future, I think it's good to unify webgpu and webnn. It seems that all other flags are lower case. Maybe it's better to use lower case. No strong preference myself.